src/OgGroupAudienceHelper.php

```diff
@@ -56,4 +57,58 @@ public static function checkFieldCardinality(ContentEntityInterface $entity, $fi
     return $entity->get($field_name)->count() < $cardinality;
   }
 
+  /**
+   * Returns the first group audience field that matches the given entity.
+   *
+   * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+   *   The group content to find a matching group audience field for.
+   * @param string $group_type
+   *   The group type that should be referenced by the group audience field.
+   * @param string $group_bundle
+   *   The group bundle that should be referenced by the group audience field.
+   * @param bool $check_access
+   *   Set this to FALSE to not check if the current user has access to the
+   *   field. Defaults to TRUE.
+   *
+   * @return string|NULL
+   *   The name of the group audience field, or NULL if no matching field was
+   *   found.
+   */
+  public static function getMatchingField(ContentEntityInterface $entity, $group_type, $group_bundle, $check_access = TRUE) {
+    $fields = Og::getAllGroupAudienceFields($entity->getEntityTypeId(), $entity->bundle());
+
+    // Bail out if there are no group audience fields.
+    if (!$fields) {
+      return NULL;
```

Comment:
@pfrenssen in your original PR you added a comment about maybe returning an empty string. Is there a good case to why it would be more right than null?